test/profile/Makefile

```diff
@@ -33,7 +33,7 @@ $(ROOT)/both.done: $(ROOT)/%.done: $(ROOT)/%
 	$(QUIET)$(TIMELIMIT)$(ROOT)/$* $(ROOT)/both.log $(ROOT)/both.def $(ROOT)/bothgc.log
 	$(QUIET)$(GREP) -q '1 .*_Dmain' $(ROOT)/both.log
 	$(QUIET)$(GREP) -q '1000 .*both.Num\* both.foo(uint)' $(ROOT)/both.log
-	$(QUIET)$(DIFF) both.def.exp $(ROOT)/both.def
+	$(QUIET)$(DIFF) both.def.exp $(ROOT)/both.def || $(DIFF) both.def.exp2 $(ROOT)/both.def
```

Comment:
Maybe adding a new comment that this allows the old & new mangling and can be removed in the future wouldn't hurt?